scripts/run_validator_node.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+set -ue
+set -o pipefail
+
+sudo_cmd=""
+files_path=""
```

Comment:
What kind of files? Variable name is too general.

Reply:
"files_path" specifies the directory where all the node's data is stored